pkg/cloudprovider/cloudprovider.go

```diff
@@ -254,6 +254,11 @@ func (c *CloudProvider) RepairPolicies() []cloudprovider.RepairPolicy {
 			ConditionStatus:    corev1.ConditionFalse,
 			TolerationDuration: 30 * time.Minute,
 		},
+		{
+			ConditionType:      corev1.NodeReady,
+			ConditionStatus:    corev1.ConditionUnknown,
+			TolerationDuration: 30 * time.Minute,
```

Comment:
can we make it configurable?

Reply:
We won't make it configurable at the moment, but this is something we are open to considering in the future: https://github.com/kubernetes-sigs/karpenter/blob/main/designs/node-repair.md